lib/cfme/cloud_services/inventory_sync.rb

```diff
@@ -29,6 +29,27 @@ def self.sync_queue(userid, targets)
         MiqTask.generic_action_with_callback(task_opts, queue_opts)
       end
 
+      def self.collect(opts)
+        path = DataPackager.package(DataCollector.collect(opts[:manifest], targets_from_queue(opts[:targets])))
```

Comment:
Can we do the opts with kwargs instead?

```ruby
def self.collect(manifest:, targets:, task_id: nil)
end
```

Reply:
I tried that first actually and MiqQueue.deliver wasn't working but I mistakenly assumed it was because of the kwargs.  Turns out it was something else so yeah going back to kwargs